Makefile

```diff
@@ -15,13 +15,24 @@
 # Make will use bash instead of sh
 SHELL := /usr/bin/env bash
 
+OPA_IMAGE := opa
+OPA_IMAGE_TAG := 0.16.2
+OPA_IMAGE_URL := openpolicyagent
+
 # The .PHONY directive tells make that this isn't a real target and so
 # the presence of a file with that name won't cause this target to stop
 .PHONY: test
 test: ## Test constraint templates via OPA
 	@echo "Running OPA tests..."
 	@opa test --timeout 30s -v lib/ validator/ --explain=notes
 
+.PHONY: docker_test
+docker_test: ## Run tests using OPA docker image
+	docker run -it --rm \
```

Comment:
can you adjust this to run tests using opa 0.15.x and .016.2

Reply:
Updated using OPA image to run tests on 0.15.1 + 0.16.2 by running `make docker_test`.